.travis.yml

```
@@ -134,6 +134,13 @@ matrix:
           stage: Initial tests
           env: TOXENV='codestyle'
 
+        # ASDF schema tests
+        - language: python
+          python: 3.7
+          name: ASDF schema tests
+          stage: Initial tests
```

Since you don't have a strong opinion, why don't we not run it unless initial stage passes?
```suggestion
          stage: Comprehensive tests
```

---

This change is included in the latest commit